integration/maptest/map.go

```diff
@@ -646,6 +646,16 @@ func RunInclusionBatch(ctx context.Context, t *testing.T, tadmin trillian.Trilli
 	}
 }
 
+// RunWriteBatchStress performs checks on Trillian Map's SetLeaves call.
+func RunWriteBatchStress(ctx context.Context, t *testing.T, tadmin trillian.TrillianAdminClient, tmap trillian.TrillianMapClient, batchSize int, numBatches int) {
```

Comment:
You need to add this to AllTests at the top of this file or it won't run. But nobody would fall into that bear trap.

Reply:
Just call me "Stumpy"